README.md

```diff
@@ -1,12 +1,12 @@
-# [Qase](https://qase.io) Python
+# [Qase TestOps](https://qase.io) reporters for Python
 
-This is a monorepo for all Python-related projects
+Monorepo with [Qase TestOps](https://qase.io) reporters for Python testing frameworks.
 
 ## Projects
 
-- [qaseio](/qaseio) - Qase TestOps API client (v1)
-- [qase-python-api](/qase-python-api) - Qase TestOps API client (v2)
+- [qase-api-client](/qase-api-client) - Qase TestOps API client (v1)
```

Comment:
Let's put the user-side libs first, they're the most important items in this list.